ArduCopter/mode_guided.cpp

```diff
@@ -306,6 +316,13 @@ bool ModeGuided::set_destination(const Location& dest_loc, bool use_yaw, float y
 
 // guided_set_velocity - sets guided mode's target velocity
 void ModeGuided::set_velocity(const Vector3f& velocity, bool use_yaw, float yaw_cd, bool use_yaw_rate, float yaw_rate_cds, bool relative_yaw, bool log_request)
+{
+    Vector3f acceleration;
+    set_velaccel(velocity, acceleration, use_yaw, yaw_cd, use_yaw_rate, yaw_rate_cds, relative_yaw, log_request);
```

Comment:
It might be just as clear to pass in Vector3f() as the 2nd argument.

Reply:
I didn't think that worked but it did this time. I must be missing something....